media_driver/linux/common/os/mos_os_specific.h

```diff
@@ -179,7 +179,7 @@ typedef enum _MOS_GPU_NODE
     MOS_GPU_NODE_VIDEO   = I915_EXEC_BSD,
     MOS_GPU_NODE_VIDEO2  = I915_EXEC_VCS2,
```

Comment:
how about change value of I915_EXEC_VCS2 to 6<<0 as well?

Reply:
Changed the I915_EXEC_VCS2 to 6 << 0. don't change MOS_GPU_NODE_MAX in https://github.com/intel/media-driver/pull/1164